index.ts

```diff
@@ -613,9 +631,9 @@ commander
               if (abortPush) return spinner.stop(true);
 
               const files = filePaths.map((name, i) => {
-                let nameWithoutExt = name.slice(0, -path.extname(name).length);
-                // Formats rootDir/appsscript.json to appsscript.json. 
-                // Preserves subdirectory names in rootDir 
+                const nameWithoutExt = name.slice(0, -path.extname(name).length);
```

Comment:
This change will conflict with #48. Don't change code that's out of scope of the fix next time.

Reply:
Will keep that in mind. npm lint complained that there was let being instead of const on those two lines so I just changed it. I'll scope my PRs better next time.

Reply:
I got an error on  tslint when let was used in those lines so I changed them to const. I'll keep this in mind for future PRs.